### PR TITLE
rename og60 to tofu60

### PR DESCRIPTION
**Pull request by moyi4681**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
Missing corresponding entry in `/data/mappings/keyboard_aliases.hjson`

File: data/mappings/keyboard_aliases.hjson [hunk 1]
```diff
@@ -184,6 +184,9 @@
     },
     "dztech/volcano660": {
         "target": "ilumkb/volcano660"
+    },
+	"dztech/tofu60": {
```
Comment (dunk2k) [inline data/mappings/keyboard_aliases.hjson:188]:
```suggestion
	"dztech/og60": {
```

File: keyboards/dztech/tofu60/keymaps/default/keymap.c
Comment (dunk2k) [inline keyboards/dztech/tofu60/keymaps/default/keymap.c:1]: As layers 1-3 are repeats of layer 0 and there is no means to switch between layers, could layer count be reduced to just layer 0

File: keyboards/dztech/tofu60/keymaps/via/keymap.c
Comment (dunk2k) [inline keyboards/dztech/tofu60/keymaps/via/keymap.c:1]: As layers 1-3 are repeats of layer 0 and there is no means to switch between layers, could layer count be reduced to just layer 0

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: data/mappings/keyboard_aliases.hjson [hunk 2]
```diff
@@ -184,6 +184,9 @@
     },
     "dztech/volcano660": {
         "target": "ilumkb/volcano660"
+    },
+	"dztech/og60": {
```
Comment (fauxpark) [inline data/mappings/keyboard_aliases.hjson:188]:
```suggestion
    "dztech/og60": {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**